cmd/thriftrw-list-deps/main_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRecursiveImports(t *testing.T) {
+	outputBuf := new(bytes.Buffer)
+	cwd, err := os.Getwd()
+	require.NoError(t, err)
+	idlRoot := filepath.Join(cwd, "../../gen/testdata/thrift")
+	err = do(idlRoot, filepath.Join(idlRoot, "structs.thrift"), outputBuf)
+	require.NoError(t, err)
+	outputLines := strings.Split(outputBuf.String(), "\n")
+	// t.Logf("output lines: %+v", outputLines)
+	assert.Equal(t, 2, len(outputLines))
+	assert.Equal(t, "enums.thrift", outputLines[0])
+	assert.Equal(t, "", outputLines[1])
+
+	outputBuf.Reset()
```

Comment:
nit: Rather than a reset buffer / re-do, we could use sub-tests or table-driven tests.

i.e. something similar to: 
```
t.Run("default options", ...)
t.Run("with relative path", ...)
```

With that said, could we add a test that includes an output with multiple expected thrift files?